src/main/java/com/kodilla/ecommercee/exception/advice/ExceptionAdvice.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.ecommercee.exception.advice;
+
+import com.kodilla.ecommercee.exception.AuthorizationDeniedException;
+import com.kodilla.ecommercee.exception.EntityAlreadyExistsException;
+import com.kodilla.ecommercee.exception.EntityNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
```

Comment:
Globalna obsluga exceptionów to jak najbardziej dobry pomysł. Planowałem porozmawiać o tym poźniej, ale skoro już to zrobiłeś to możemy śmiało dodać